### PR TITLE
Add drupal/smtp to composer.json

### PR DESCRIPTION
**Pull request by baldwinlouie**
Added drupal/smtp to composer.json for Cloud Orchestrator distribution.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -114,6 +114,7 @@
     "drupal/roleassign": "^2",
     "drupal/simple_oauth": "^5",
     "drupal/tfa": "^2",
+    "drupal/smtp": "^1",
```
Comment (naoi) [inline composer.json]: @baldwinlouie The item should be the alphabetical order:

```suggestion
    "drupal/smtp": "^1",
    "drupal/tfa": "^2",
```

**Review by naoi — CHANGES_REQUESTED** [review]
@baldwinlouie I posted my comment.  Thanks!

File: composer.json (hunk 1 above)
Comment (baldwinlouie) [reply composer.json]: @naoi This has been updated.

**Comment by baldwinlouie** [comment]
@naoi Please review again.  I've updated the file.

**Review by naoi — APPROVED** [review]
@baldwinlouie Thank you for the update.  I'll merge the PR.  Thanks!

**Pull request merged**